### PR TITLE
[WebApp] Add to `az webapp deploy` param help text

### PR DESCRIPTION
**Pull request by JasonFreeberg**
String changes for the `az webapp deploy` command's parameter help text.

**Description**

Grammatical and readability fixes for the help text on the new `az webapp deploy` command

**Testing Guide**
It's just string changes

### CODE REVIEW

**Review by qwordy — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 1]
```diff
@@ -641,37 +641,37 @@ def load_arguments(self, _):
...
-        c.argument('src_url', options_list=['--src-url'], help='url to download the package from. Example: http://mysite.com/files/myapp.war?key=123')
-        c.argument('target_path', options_list=['--target-path'], help='Target path relative to wwwroot to which the file will be deployed to.')
-        c.argument('artifact_type', options_list=['--type'], help='Type of deployment requested')
+        c.argument('name', options_list=['--name', '-n'], help='Name of the function app to deploy to.')
+        c.argument('src_path', options_list=['--src-path'], help='Path of the artifact to be deployed. Ex: "myapp.zip" or "/myworkspace/apps/myapp.war"')
+        c.argument('src_url', options_list=['--src-url'], help='URL of the artifact. The webapp will pull the artifact from this URL. Ex: "http://mysite.com/files/myapp.war?key=123"')
+        c.argument('target_path', options_list=['--target-path'], help='Absolute path that the artifact should be deployed to. Defaults to "home/site/wwwroot/" Ex: "/home/site/deployments/tools/", "/home/site/scripts/startup-script.sh".')
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py:667]:
```suggestion
        c.argument('target_path', options_list=['--target-path'], help='Absolute path that the artifact should be deployed to. Defaults to "home/site/wwwroot/". Ex: "/home/site/deployments/tools/", "/home/site/scripts/startup-script.sh".')
```

**Review by qwordy — APPROVED** [review]

**Comment by JasonFreeberg** [comment]
@qwordy -- The Azure CI checks keep failing and I don't understand the error messages. Can you help me?

**Comment by qwordy** [comment]
> 
> 
> @qwordy -- The Azure CI checks keep failing and I don't understand the error messages. Can you help me?

OK. It is a rare error.

**Comment by JasonFreeberg** [comment]
@qwordy -- It's still stuck. What do we do now?

**Comment by qwordy** [comment]
@JasonFreeberg, New pull requests in this repository look good. Could you git pull from dev branch and try again?

**Comment by JasonFreeberg** [comment]
@qwordy -- Still failing on some obscure debian packaging step: https://dev.azure.com/azure-sdk/public/_build/results?buildId=903690&view=logs&j=a9d377d9-8e0e-5471-d3f2-14bad4270dd0

**Comment by qwordy** [comment]
I created a same [PR](https://github.com/Azure/azure-cli/pull/18163) to test if it can pass.

**Comment by JasonFreeberg** [comment]
Thank you @qwordy !

**Pull request merged**